scripts/update/10-dependencies.sh

```diff
@@ -6,6 +6,13 @@ if ! which add-apt-repository > /dev/null; then
 fi
 
 if [[ $(_os_distro) == "ubuntu" ]]; then
+    if [[ $(_os_codename) == "jammy" ]]; then
+        if ! grep 'ubuntu-toolchain-r' /etc/apt/sources.list | grep -q -v '^#'; then
```

Comment:
This is an invalid check. On jammy, the repo is configured to 

`/etc/apt/sources.list.d/ubuntu-toolchain-r-ubuntu-ppa-jammy.list`